Tools/AP_Periph/Parameters.h

```diff
@@ -62,6 +62,10 @@ class Parameters {
         k_param_esc_telem_rate,
         k_param_can_slcan_cport,
         k_param_temperature_sensor,
+        k_param_proximity,
+        k_param_proximity_baud,
+        k_param_proximity_port,
+        k_param_proximity_max_rate,
```

Comment:
personally I'd rename this enum to k_param_proximity_rate_max but this is optional of course.

Reply:
Again, kept consistent with rangefinder. I would be happy to change both if you would like

Reply:
it's ok thanks.